react-calculator/src/components/AC/index.tsx

```diff
@@ -0,0 +1,16 @@
+interface Props {
+  allClear: () => void;
+}
+
+const AC = (props: Props) => {
+  const { allClear } = props;
```

Comment:
객체 구조 분해를 사용하면 어떨까요?

```suggestion
const AC = ({ allClear }: Props) => {
```